README.md

```diff
@@ -1,2 +1,62 @@
 # Hello_Microverse
 Hello World
+
+
+
+
+
```

Comment:
- [ ] Kindly add a description to your project 💯

> In this project, I built Hello Microverse page. The only goal here is to master all the tools and best practices I learned in the previous steps because I will be using them on all my upcoming Microverse projects.